sdk/communication/AzureCommunicationChat/AzureCommunicationChat.xcodeproj/project.pbxproj

```diff
@@ -1309,7 +1311,7 @@
 			repositoryURL = "https://github.com/microsoft/trouter-client-ios/";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = "0.0.1-beta.4";
+				minimumVersion = "0.0.1-beta.5";
```

Comment:
I think this is a branch, not a tag, according to Jiri.

Reply:
its tagged!